site/source/_posts/2019/01/28/Version-5-Dojo.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Dojo 5
+date: 2019-01-28 08:00:00
+authorId: agubler
+featured_image: featured.jpg
+---
+
+Dojo version 5 has arrived!
+
+We're excited to announce the 5.0.0 release of Dojo. This version builds on previous releases with a vast array of bug fixes and improvements throughout the framework.
+
+{% asset_img featured.jpg feature-full %}
+
+### Conditional Polyfills - Serving Less JavaScript
+
+One of our primary goals for modern Dojo is to improve performance by serving minimal JavaScript bundles by default. For version 5, we're happy to announce an out of the box solution to optimize the user experience for dealing with bundling and loading polyfills in Dojo applications.
+
+The Dojo build produces separate "platform" bundles that will be **only** be loaded based on two key conditions:
+
+1) The shim module is imported somewhere in an application.
+1) A users browser does not natively support the browser feature.
+
+In short, if your application does not use the browser feature or you use a browser with native support, the polyfill will not be loaded! This change means serving less JavaScript and improving your application performance without compromising on features.
+
+### New Polyfills Added to Shim
+
+We've added additional third party polyfills to `@dojo/framework/shim`:
+
+* [`@dojo/framework/shim/fetch`][FetchAPI] - The Fetch API provides an interface for fetching resources (including across the network).
+* [`@dojo/framework/shim/IntersectionObserver`][IntersectionObserverAPI] - The Intersection Observer API provides a way to asynchronously observe changes in the intersection of a target element with an ancestor element or with a top-level document's viewport.
+* [`@dojo/framework/shim/WebAnimations`][WebAnimationsAPI] - The Web Animations API allows for synchronizing and timing changes to the presentation of a Web page, i.e. animation of DOM elements.
+* [`@dojo/framework/shim/ResizeObserver`][ResizeObserverAPI] - The ResizeObserver interface reports changes to the content rectangle of an Element or the bounding box of an SVGElement. The content rectangle is the box in which content can be placed, meaning the border box minus the padding.
+
+To use these additional polyfills they should be imported like any other ponyfill provided by `@dojo/framework/shim` and they no longer need to be explicitly added/import to your Dojo application.
```

Comment:
```suggestion
To use these additional polyfills they should be imported like any other ponyfill provided by `@dojo/framework/shim` and they no longer need to be explicitly added/imported to your Dojo application.
```

Also it might be confusing that we use polyfill and ponyfill here?